modules/oe_content_event/config/install/field.field.node.oe_event.field_featured_media_legend.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_featured_media_legend
+    - node.type.oe_event
+id: node.oe_event.field_featured_media_legend
+field_name: field_featured_media_legend
+entity_type: node
+bundle: oe_event
+label: 'Featured media legend'
+description: ''
+required: false
```

Comment:
should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Description fields group filled in.